test_requirements.txt

```diff
@@ -1,10 +1,29 @@
-setuptools
+atomicwrites
```

Comment:
This list looks excessive. We certainly don't use most of them directly in the tests, and for dependencies of  dependencies, pip should take care of installing those (especially since they will change over time).

Reply:
I have python 3.6 so all requirements is downlaoded itself in virtualenv so I run `pip freeze > test_requirements.txt`. I am not aware of these dependencies of dependencies.